x/datadeal/keeper/deal_test.go

```diff
@@ -617,3 +617,58 @@ func (suite dealTestSuite) TestDataDeliveryVoteFaildInvalidStatus() {
 	err = suite.DataDealKeeper.VoteDataDelivery(ctx, dataDeliveryVote, signature)
 	suite.Require().ErrorIs(err, types.ErrDataDeliveryVote)
 }
+
+func (suite dealTestSuite) TestDeactivateDeal() {
+	ctx := suite.Ctx
+
+	_ = suite.MakeTestDeal(1, suite.buyerAccAddr)
```

Comment:
There is no need to make test deal for this test. In `BeforeTest()`, a test deal (dealID of 1) is set. So we can just deactivate the deal.
```suggestion
```